.vscode/settings.json

```diff
@@ -48,6 +48,14 @@
     "env": {
       "LANG": "en-US",
       "TZ": "UTC",
+
+      // Uncomment to set a custom path to a CodeQL checkout.
+      // "TEST_CODEQL_PATH": "../codeql",
```

Comment:
Do we still need this in [launch.json](https://github.com/github/vscode-codeql/blob/main/.vscode/launch.json#L147)?

Reply:
Yes, the `launch.json` is used when you run the "Launch Integration Tests - With CLI" action using the "Run & Debug" panel of VSCode. These environment variables are instead used when you use the Jest runner to run tests. These are completely separate.